libcob/ChangeLog

```diff
@@ -543,6 +564,11 @@
 
 	* reportio.c: Fixed spacing and test cases to match Microfocus
 
+2021-12-22  Simon Sobisch <simonsobisch@gnu.org>
+
+	* fileio.c (indexed_close) [WITH_DB]: fix bug #753 sigsegv on CLOSE
+	  when COB_SYNC is active
```

Comment:
Are you sure that was merged to fbdb.c correctly?

Reply:
The patch (4709) just adds `f->file = NULL;` after `cob_free (p);` in `ix_bdb_close` ; this has been correctly merged (+ mergeinfo updated specifically). Does anything make you think otherwise ?
    `

Reply:
no, that's fine, I just wondered; thanks for also setting mergeinfo appropriately